presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGeneratorSql.java

```diff
@@ -13,7 +13,18 @@
  */
 package com.facebook.presto.pinot.query;
 
+import com.facebook.presto.common.block.SortOrder;
+import com.facebook.presto.spi.plan.AggregationNode;
+import com.facebook.presto.spi.plan.Assignments;
+import com.facebook.presto.spi.plan.Ordering;
+import com.facebook.presto.spi.plan.OrderingScheme;
+import com.facebook.presto.spi.plan.ProjectNode;
+import com.facebook.presto.spi.plan.TableScanNode;
+import com.facebook.presto.spi.plan.TopNNode;
+import com.facebook.presto.sql.planner.iterative.rule.test.PlanBuilder;
 import com.facebook.presto.testing.assertions.Assert;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import org.testng.annotations.Test;
 
 public class TestPinotQueryGeneratorSql
```

Comment:
Can you point me to where the mixed group-by-aggregation projection tests are added ? I couldn't see them in this batch of changes. The closes that it comes to is `testAggregationWithGroupByOrderBy`. Its important to test those particularly since we switched from `expectedColumnIndices` to `outputs` way of handling it.

Reply:
testAggregationWithGroupBy() has tests for aggregation-groupbys, using AggregationNode and ProjectNode.

testAggregationWithOrderByPushDownInTopN() has tests for aggregation-groupbys-orderbys using AggregationNode and TopNNode.